distribution/scripting.md

```diff
@@ -38,6 +40,7 @@ registerPlugin({
     version: '1.0',
     authors: ['Your Name'],
     type: 'remote',
+    license: 'MIT',
```

Comment:
The code is using `licence`, but the example uses `license`. We should probably support both, but:
```suggestion
    licence: 'MIT',
```